generate-text-packs.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+set -e
+
+usage()
+{
+	echo "Usage: $0 --name <package_name> --versions <package_versions> [options]"
+	echo ""
+	echo "  Restores specified package versions and generates a source-build-usable text-only package."
+	echo "  The resulting text-only packages can be found in the './src/textOnlyPackages/src' directory."
+	echo ""
+	echo "  Both --name and --versions must be specified"
+	echo ""
+	echo "  --name <package_name>          Name of the package. Example: Microsoft.Docker.Sdk (short: -n)"
+	echo "  --versions <package_versions>  Versions of the package, separated by a comma. Example: 1.1.0,1.0.0 (short: -v)"
+	echo ""
+	echo "Options:"
+	echo "  --feeds \"<value>\"            Semicolon-separated list of additional Nuget feeds to use during restore (short: -f)"
+}
+
+source="${BASH_SOURCE[0]}"
+
+# resolve $SOURCE until the file is no longer a symlink
+while [[ -h $source ]]; do
+  scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
+  source="$(readlink "$source")"
+
+  # if $source was a relative symlink, we need to resolve it relative to the path where the
+  # symlink file was located
+  [[ $source != /* ]] && source="$scriptroot/$source"
+done
+
+scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
+
+package_name=''
+package_versions=''
+feeds=''
+
+if [[ $# -le 0 ]]; then
+	usage
+	exit 0
+fi
+
+while [[ $# > 0 ]]; do
+	opt="$(echo "${1/#--/-}" | tr "[:upper:]" "[:lower:]")"
```

Comment:
Linux command options are commonly case sensitive, we shouldn't be manipulating case of the options.